mslib/utils/airdata.py

```diff
@@ -79,19 +82,22 @@ def download_progress(file_path, url, progress_callback=lambda f: logging.info(f
         QtWidgets.QMessageBox.information(None, "Download failed", f"{url} was unreachable, please try again later.")
 
 
-def get_airports(force_download=False):
+def get_airports(force_download=False, url=None):
     """
     Gets or downloads the airports.csv in ~/.config/msui and returns all airports within
     """
     global _airports, _airports_mtime
-    file_exists = os.path.exists(os.path.join(MSUI_CONFIG_PATH, "airports.csv"))
+    if url is None:
```

Comment:
a string is not a list. why not add the default string in the function declaration?